src/client/cli/cmd/launch.cpp

```diff
@@ -378,10 +378,11 @@ mp::ParseCode cmd::Launch::parse_args(mp::ArgParser* parser)
 
     if (parser->isSet(cloudInitOption))
     {
+        constexpr auto err_msg_template = "Could not load cloud-init configuration from '{}': {}\n";
```

Comment:
Is the '\n' at the end necessary? You are using this template in a `println` which adds its own new line, so as a result there is an extra empty line in the terminal output.

Reply:
Ah, of course.